src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -172,7 +172,10 @@ xorGate i1 i2 o = [rise i1, rise i2] ~|~> rise o <> [fall i1, fall i2] ~|~> rise
                <> [rise i1, fall i2] ~|~> fall o <> [fall i1, rise i2] ~|~> fall o
 
 srLatch :: Eq a => a -> a -> a -> CircuitConcept a
-srLatch s r q = complexGate ((Var s) `And` (Not (Var r))) ((Var r) `And` (Not (Var s))) q
+srLatch s r q = complexGate (Var s) (Var r) q
+
+srLatch2 :: Eq a => a -> a -> a -> a -> CircuitConcept a
+srLatch2 s r q nq = srLatch s r q <> bubble nq (srLatch s r nq)
 
```

Comment:
@snowleopard `dual` didn't quite work as well for `nq`. But `bubble bq` is the exact operation we desire.